docs/contributing-analyzer.md

```diff
@@ -99,26 +99,6 @@ If you run the project `ReviewDiffs` in debug mode, it will print in the output
 
 *PS*: This program outputs message in a way that is compatible with the [VsColorOutput](https://marketplace.visualstudio.com/items?itemName=MikeWard-AnnArbor.VSColorOutput) extension, so that errors are colored in red, and differences are colored like warnings.
 
-### Manual Tests
-
-From Visual Studio, make sure `SonarAnalyzer.Vsix.csproj` is selected as startup project. And then do the following:
```

Comment:
should we replace with some other instructions to debug analysis of a project (e.g. setting a `Debugger.Launch()`) ?

Reply:
Yes, that would be great.